scim/data_service_principal_test.go

```diff
@@ -56,7 +56,7 @@ func TestDataServicePrincipalReadNotFound(t *testing.T) {
 		Fixtures: []qa.HTTPFixture{
 			{
 				Method:   "GET",
-				Resource: "/api/2.0/preview/scim/v2/ServicePrincipals?filter=applicationId%20eq%20%27abc%27",
+				Resource: "/api/2.0/preview/scim/v2/ServicePrincipals?excludedAttributes=roles&filter=applicationId%20eq%20%27abc%27",
```

Comment:
For single service principal/user - keep roles, it’s part of the public APIs

Reply:
We do not have output attributes that depend read roles for users/service principals. For groups we do, which is the instance_profiles. Doing a second pass on this just to be sure

Reply:
So we use the list user API in three instances:
1. Create user (resource): We only use the Id here
2. Read user (data source): we use UserName, DisplayName, ExternalID, ApplicationID, ID here
3. Exporter:  UserName, DisplayName, Id, Groups

Does not use roles